src/lib/theme.ts

```diff
@@ -10,6 +10,7 @@ export const styles = {
             transition duration-200`,
         sm: `px-3 py-1.5 text-sm`,
         md: `px-4 py-2 text-base`,
+        highlighted: `bg-gray-700/80 border-gray-500`,
```

Comment:
I did that because I couldn't find a way to use the "active" CSS defined above with classes, maybe there is a way?

Reply:
Looks fine to me tbh.

Active would imply a pressed state, not selected/highlighted.